### PR TITLE
Add_globalazurevirtualkorea

### PR DESCRIPTION
**Pull request by kimsejun2000**
Global Azure Virtual 2022 on Korea will be hosted by Korea Azure User Group and Wise Azure Life.
This event is a local event and sessions will be held in Korean.
4 tracks and 17 speakers.

### CODE REVIEW

**Comment by kimsejun2000** [comment]
Links linked are temporary. It will be updated soon.

File: 2022/globalazurekorea/data.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "kimsejun2000"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Global Azure Virtual 2022 in Korea",
+    "Country": "Seoul",
```
Comment (noopman) [inline 2022/globalazurekorea/data.json:10]: Seoul is a city. Please put country!

**Review by noopman — CHANGES_REQUESTED** [review]
Just a tiny change needed. All else looks very good!

**Comment by noopman** [comment]
> Global Azure Virtual 2022 on Korea will be hosted by Korea Azure User Group and Wise Azure Life. This event is a local event and sessions will be held in Korean. 4 tracks and 17 speakers.

THAT is AWESOME! Can't wait to get you on the site and I will make lots of noise about it too! Welcome to Global Azure OUR league of communities!

**Comment by kimsejun2000** [comment]
I modified Country with your help.

**Review by noopman — APPROVED** [review]
That's perfect @kimsejun2000 merging your request now! Welcome to the league of extraordinary global azure communities! :D

**Pull request merged**